### DISCUSSION TITLE
from 720kB to 1.44Mb

### DISCUSSION THREAD

**Discussion by linklocal** [keirf/flashfloppy#473] [General]
maybe following question already arise, but i didn't found anything about.

i plan to use gotek hardware to replace a floppy dive inside a AGILENT 546xx scope. but cannot purchase a 1.44MB "SFR1M44-DU26 GoTek 26Pin FPC 1.44M" hardware. 

available is the 720kb counterpart: "SFRM72-DU26 720K USB SSD". i'm not familiar with the interface , but it looks the same. is it possible to reflash this and get the abilities from the 1.44Mb-drive? 

best regards

**Comment by keirf** [comment]
Yes, FlashFloppy will work on that drive just fine and support both 720k and 1.44M in the single firmware.

**Comment by linklocal** [comment]
thank you for comment. as soon the drive HW arrive i will check.

so far ... really good work.
